src/main.rs

```diff
@@ -369,18 +359,31 @@ mod tests {
             .mount(&mock_server)
             .await;
 
+        Mock::given(method("POST"))
+            .and(path("/network-subgraph"))
+            .respond_with(ResponseTemplate::new(200).set_body_string(
+                r#"{
+                    "data": {
+                        "block_hash_from_number" : "193bb3a5e78b8726f6138cfae7dd18c83fe32843032999966fd2ca6973f88f3b"
+                    },
+                    "errors": null
+                }"#,
+            ))
+            .mount(&mock_server)
+            .await;
+
         let private_key = env::var("PRIVATE_KEY").expect("No private key provided.");
-        let eth_node = env::var("ETH_NODE").expect("No ETH URL provided.");
+        let _eth_node = env::var("ETH_NODE").expect("No ETH URL provided.");
 
         // TODO: Add something random and unique here to avoid noise form other operators
         let radio_name: &str = "test-poi-crosschecker-radio";
 
         let graphcast_agent = GraphcastAgent::new(
             private_key,
-            eth_node,
             radio_name,
             &(mock_server.uri() + "/graphcast-registry"),
             &(mock_server.uri() + "/network-subgraph"),
+            &(mock_server.uri() + "/graph-node-status"),
```

Comment:
shouldn't this be `graphql`?

Reply:
If the tests post at the same endpoint, then don't think this really matter?